webvtt.html

```diff
@@ -542,6 +542,17 @@
    </section>
 
    <section>
+    <h3>Unicode normalization</h3>
+
+    <p>This specification does not normalize Unicode text as part of its processing
+    requirements.</p>
+
+    <p>For example, a cue with the identifier consisting of the LATIN CAPITAL LETTER A followed by
```

Comment:
David: As a UA dev, you are required not to normalize, because the spec does not normalize. Therefore, as a Web dev, don't expect the UA to normalize.

Would have some better wording to describe this?